docs/tutorials/01-intro.md

```diff
@@ -0,0 +1,209 @@
+
+    buildah push fedora-bashecho docker-daemon:fedora-bashecho:latest
+
+Under the covers, the containers/image library calls into the containers/storage library to read the image's contents, and sends them to the local docker daemon. This can take a little while. And usually you won't need to do this. If you're using `buildah` you are probably not using docker. This is just for demo purposes. Let's try it:
+
+    docker run fedora-bashecho 
+    This is a new container from ipbabble [ 0 ]
+    This is a new container from ipbabble [ 1 ]
+    This is a new container from ipbabble [ 2 ]
+    This is a new container from ipbabble [ 3 ]
+    This is a new container from ipbabble [ 4 ]
+    This is a new container from ipbabble [ 5 ]
+    This is a new container from ipbabble [ 6 ]
+    This is a new container from ipbabble [ 7 ]
+    This is a new container from ipbabble [ 8 ]
+    This is a new container from ipbabble [ 9 ]
+    
+OCI container images built with `buildah` are completely standard as expected. So now it might be time to run:
+
+    dnf -y remove docker
+
+## Using Dockerfiles with buildah
+
+What if you have been using docker for while and have some existing Dockerfiles. Not a problem. `buildah` can build images using a Dockerfile. The `build-using-dockerfile`, or `bud` for short, takes a Dockefile as input and produces an OCI image.
+
+Find one of your Dockerfiles or create a file called Dockerfile. Use the following example or some variation if you'd like:
+
+    # Base on the Fedora
+    FROM fedora:latest
+    MAINTAINER W Henry email ipbabble@gmail.com
+
+    # Update image and install httpd
+    RUN echo "Updating all fedora packages"; dnf -y update; dnf -y clean all
+    RUN echo "Installing httpd"; dnf -y install httpd
+ 
+    # Expose the default httpd port 80
+    EXPOSE 80
+
+    # Run the httpd
+    CMD ["/usr/sbin/httpd", "-DFOREGROUND"]
+
+Now run `buildah bud` with the name of the Dockerfile and the image name (fedora-httpd):
+
+    buildah bud -f Dockerfile -t fedora-httpd
+
+You will see all the steps of the Dockerfile executing. Afterwards `buildah images` will show you the new image. Now we need to create the container and test it with `buildah run` it:
+
+    httpcontainer=$(buildah from fedora-httpd)
+    buildah run $httpcontainer
+    
+While that container is running, in another shell run:
+
+    curl localhost
+    
+You will see the standard apache webpage.
+
+Why not try and modify the Dockerfile. Do not install httpd, but instead ADD the runecho.sh file and have it run as the CMD. 
```

Comment:
Summation?  Might be nice to add some pointers where they can get more info, thank them for their time, hope they'll contribute, etc., etc., etc.

Reply:
Ack. Again I don't think that's needed right away. Can I (or you) add it later as a patch to this file or does it need to be there now? Also that would be important for the public blog - so it needs to be done for that for sure. 

Or I can fix it now while the build is still broken and resubmit. But I'll need help on the git process. I'm sure it's more than just a git commit --amend -s ... I.e. come checkout, change and commit on my branch. (?)

Reply:
I'll send my super secret set of git commands to you in email.  Hopefully they'll ease the pain.

Reply:
Added a Congratulations section.